typing/env.ml

```diff
@@ -3075,9 +3081,17 @@ let report_lookup_error _loc env ppf = function
   | Unbound_class lid ->
       fprintf ppf "Unbound class %a" !print_longident lid;
       spellcheck ppf extract_classes env lid;
-  | Unbound_modtype lid ->
+  | Unbound_modtype lid -> begin
       fprintf ppf "Unbound module type %a" !print_longident lid;
       spellcheck ppf extract_modtypes env lid;
```

Comment:
Do we want this new hint before, or after, the spellchecking hint (if any)? (I'm not sure if one order is objectively better than the other.)